app/controllers/sign_up/registrations_controller.rb

```diff
@@ -1,5 +1,5 @@
 module SignUp
-  class RegistrationsController < Devise::RegistrationsController
+  class RegistrationsController < ApplicationController
```

Comment:
If the tests are still passing, doesn't that mean that it is OK to remove this?

My thinking behind moving us away from Devise inheritance in controllers is that it makes it very difficult to know what is going on. It's one thing if we just used Devise, but we are overriding so many methods that figuring out what we control and what they control is really hard. If we no longer inherit from Devise, then we know that all of the controller code being used is in our app.